waspc/data/Generator/templates/sdk/wasp/server/_types/serialization.ts

```diff
@@ -20,8 +20,8 @@ type PrimitiveJSONValue = string | number | boolean | undefined | null
 
 export interface JSONArray extends Array<JSONValue> {}
 
-type SerializableJSONValue =
-  | Symbol
+export type SerializableJSONValue =
+  | symbol
```

Comment:
Just fixing a TS linting warning here, it's because of the same reason why we shouldn't use `String` instead of `string`

Reply:
Yeah, seems right to me...

But I copied this over from superjson (link in the comment above). And they still have `Symbol`. Can you open an issue there and see whether we're missing something?

Or you'll help them find a bug - win win :smile: 

Oh, and what threw this error? I'm not getting anything.

Reply:
I opened a PR, let's see if they have any complaints: https://github.com/flightcontrolhq/superjson/pull/318
The TypeScript Handbook explicitly calls this out so I'm not sure there's a reason for not merging it: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

> Oh, and what threw this error? I'm not getting anything.
When I wrote this it was because I though I got the linting warning. Apparently the linter was my own brain 'cause I can't get it again 😂